snippets/lvms-disconnected-ImageSetConfig.adoc

```diff
@@ -25,12 +25,12 @@ mirror:
   - name: registry.redhat.io/ubi9/ubi:latest <9>
   helm: {}
 ----
-<1> Add `archiveSize` to set the maximum size, in GiB, of each file within the image set.
-<2> Set the back-end location to save the image set metadata to. This location can be a registry or local directory. It is required to specify `storageConfig` values, unless you are using the Technology Preview OCI feature.
-<3> Set the registry URL for the storage backend.
-<4> Set the channel to retrieve the {product-title} images from.
-<5> Add `graph: true` to generate the OpenShift Update Service (OSUS) graph image to allow for an improved cluster update experience when using the web console. For more information, see _About the OpenShift Update Service_.
-<6> Set the Operator catalog to retrieve the {product-title} images from.
-<7> Specify only certain Operator packages to include in the image set. Remove this field to retrieve all packages in the catalog.
-<8> Specify only certain channels of the Operator packages to include in the image set. You must always include the default channel for the Operator package even if you do not use the bundles in that channel. You can find the default channel by running the following command: `oc mirror list operators --catalog=<catalog_name> --package=<package_name>`.
-<9> Specify any additional images to include in image set.
\ No newline at end of file
+<1> Set the the maximum size (in GiB) of each file within the image set.
+<2> Specify the location in which you want to save the image set. This location can be a registry or a local directory. You must configure the `storageConfig` field unless you are using the Technology Preview OCI feature.
```

Comment:
@jakobmoellerdev @suleymanakbas91 
Could you please specify the TP feature that is mentioned in this line?

Thanks!

Reply:
I guess this information was taken from https://docs.openshift.com/container-platform/4.12/installing/disconnected_install/installing-mirroring-disconnected.html and is not valid for 4.13 and above. We can remove the part after unless.

Reply:
Or we could just add a reference to that page instead of duplicating it?

Reply:
Hi @suleymanakbas91 
Thanks for the confirmation!
Updated as per the suggestion. 

I was thinking adding a link to the "Image set configuration parameters" module in the Addn. resources section. And keep the snippet (description of parameters) as is because the "packages.name" field for installing LVMS is different from the example provided in the "Creating the image set configuration" section. 

WDYT?